.coderabbit.yaml

```diff
@@ -0,0 +1,81 @@
+# Docs: https://docs.coderabbit.ai/configure-coderabbit
+# Schema: https://coderabbit.ai/integrations/schema.v2.json
+
+language: en
+
+tone_instructions: |
+  Provide feedback in a professional, friendly, constructive, and concise tone.
+  Offer clear, specific suggestions and best practices to help enhance the code quality and promote learning.
+
+early_access: true
+
+knowledge_base:
+  # The scope of learnings to use for the knowledge base.
+  # `local` uses the repository's learnings,
+  # `global` uses the organization's learnings,
+  # `auto` uses repository's learnings for public repositories and organization's learnings for private repositories.
+  # Default value: `auto`
+  learnings:
+    scope: global
+  issues:
+    scope: global
+  pull_requests:
+    scope: global
+
+reviews:
+  profile: chill
+  auto_review:
+    # Ignore reviewing if the title of the pull request contains any of these keywords (case-insensitive)
+    ignore_title_keywords:
+      - wip
+      - draft
+      - test
+  commit_status: false
+  review_status: false
```

Comment:
What do these do? We may want to add comments to most of their settings if we're trying to build a unified "Masterpoint" CR config that we'll use in all repos.

Reply:
Added comments 👍 

This is to avoid status comments like this: "Currently processing new changes in this PR. This may take a few minutes, please wait...", see the screenshot: 

<img width="624" alt="Screenshot 2024-11-13 at 15 53 02" src="https://github.com/user-attachments/assets/73533ce3-520f-4c08-8edf-6fcbc19ab243">

@coderabbit am I right?

Reply:
This is a cool back and forth. Sounds great -- Thanks for clarifying @gberenice!